### ISSUE TITLE
Account field not populated when a Contact is Created by Contact Matching in Salesforce org with Standard Architecture

### ISSUE THREAD

**Issue by jimhubert** [SFDO-Community/Summit-Events-App#194]
Summit Events does not populate the Account field when new Contacts are created by Contact Matching.  This is because the architectures used by the early adopters of Summit Events such as St. Thomas (Enrollment RX), Crown College (TargetX), and the personal sandboxes of our developers (EDA or NPSP) all populate the Account field. However as we discovered during the September 2020 Sprint when Summit Events is installed in a standard Salesforce architecture org without one of these apps then an Account value is not added to the Contact.

These Contacts that aren't associated with an Account record are called "Private Contacts" because they are always private, regardless of your organization’s sharing model. The downside of Private Contacts is that they're easy to forget because they're hidden from all users except their owner and the system administrator (per https://help.salesforce.com/articleView?id=contacts_private.htm&type=5).

We need a way to tell Summit Events that an Account value is needed for Contacts and to define what this Account value should be when a standard Salesforce architecture is used.  I believe the easiest solution is to add a record to the Custom Metadata mappings for the "Account" field on the Contact object and map a hardcoded value to a specific Account record (such as an "Unknown Account").  This could be handled through documentation and wouldn't require any coding changes.

**Comment by jimhubert** [comment]
@rdblake21 @tcdahlberg We talked about this issue that we experienced for the first time at the September Sprint of the Account field not being populated, so I added it as an issue here.  The good news is that I believe I've identified an easy solution which is to add a record to the Custom Metadata mappings for the "Account" field on the Contact object and map a hardcoded value to a specific Account record (such as an "Unknown Account"), this way the Contact Matching method will populate the Account field when the user doesn't have an app such as EDA, NPSP, EnrollmentRX, or TargetX to manage the Account for them.  This solution could be handled through documentation and wouldn't require any coding changes.
